`yarn build-static-viz:watch` errors

### Describe the bug

The `yarn build-static-viz:watch` dev script throws errors.

### To Reproduce

Checkout latest `master` branch, run `yarn build-static-viz:watch` in `metabase` directory.

### Expected behavior

There should not be any errors, and the static viz bundle should be rebuilt after any changes to files in the `frontend/src/metabase/static-viz` directory are made.

### Logs

```
work@MBP metabase % yarn build-static-viz:watch
yarn run v1.22.19
$ webpack --config webpack.static-viz.config.js --watch
assets by status 3.6 MiB [cached] 1 asset
orphan modules 2.28 MiB [orphan] 1080 modules
runtime modules 1.31 KiB 6 modules
built modules 3.58 MiB [built] 219 modules

WARNING in ../metabase-lib/references.ts 58:11-20
export 'normalize' (imported as 'normalize') was not found in 'metabase-lib/queries/utils/normalize' (module has no exports)
 @ ../metabase-lib/queries/utils/get-column-key.ts 1:0-235 6:15-35 9:7-32 14:13-44 16:6-22 16:36-57 16:71-93 17:15-40 20:20-36 20:50-68 20:82-104
 @ ./static-viz/lib/format.ts 18:0-73 79:154-166 94:186-198 108:157-169 127:155-167
 @ ./static-viz/components/RowChart/RowChart.tsx 9:0-126 35:29-58 43:24-48 47:23-42
 @ ./static-viz/components/RowChart/index.ts 2:0-37 2:0-37
 @ ./static-viz/containers/StaticChart/StaticChart.tsx 8:0-63 44:31-39
 @ ./static-viz/containers/StaticChart/index.js 1:0-40 1:0-40
 @ ./static-viz/index.js 2:0-51 5:64-75

ERROR in ../metabase-lib/queries/utils/normalize.js 1:0-50
Module not found: Error: Can't resolve 'cljs/metabase.mbql.js' in '/Users/work/Metabase/metabase/frontend/src/metabase-lib/queries/utils'
 @ ../metabase-lib/references.ts 16:0-65 58:11-20
 @ ../metabase-lib/queries/utils/get-column-key.ts 1:0-235 6:15-35 9:7-32 14:13-44 16:6-22 16:36-57 16:71-93 17:15-40 20:20-36 20:50-68 20:82-104
 @ ./static-viz/lib/format.ts 18:0-73 79:154-166 94:186-198 108:157-169 127:155-167
 @ ./static-viz/components/RowChart/RowChart.tsx 9:0-126 35:29-58 43:24-48 47:23-42
 @ ./static-viz/components/RowChart/index.ts 2:0-37 2:0-37
 @ ./static-viz/containers/StaticChart/StaticChart.tsx 8:0-63 44:31-39
 @ ./static-viz/containers/StaticChart/index.js 1:0-40 1:0-40
 @ ./static-viz/index.js 2:0-51 5:64-75

ERROR in ../metabase-lib/types/constants.js 5:0-56
Module not found: Error: Can't resolve 'cljs/metabase.types' in '/Users/work/Metabase/metabase/frontend/src/metabase-lib/types'
 @ ../metabase-lib/types/utils/isa.js 9:0-202 28:19-26 31:19-26 38:23-39 110:28-36 110:38-46 110:48-58 110:60-71 110:73-84 110:86-93 110:95-101 110:103-114 110:116-122 118:43-51 119:46-52 120:46-53 121:46-53 122:45-51 123:47-55 124:44-49 125:47-55 126:47-55 140:43-52 151:37-48 153:32-43 162:37-46 164:32-41 169:103-114 177:37-46 179:32-41 183:43-55 186:43-52 189:43-53 192:43-55 195:43-55 198:43-58 201:43-56 204:43-57 207:43-56 210:43-59 213:43-55 222:43-51 225:43-53 228:43-57 231:43-56
 @ ./static-viz/lib/format.ts 16:0-97 40:13-19 44:13-19 48:42-54 51:42-50 59:43-52
 @ ./static-viz/components/RowChart/RowChart.tsx 9:0-126 35:29-58 43:24-48 47:23-42
 @ ./static-viz/components/RowChart/index.ts 2:0-37 2:0-37
 @ ./static-viz/containers/StaticChart/StaticChart.tsx 8:0-63 44:31-39
 @ ./static-viz/containers/StaticChart/index.js 1:0-40 1:0-40
 @ ./static-viz/index.js 2:0-51 5:64-75

ERROR in ../metabase-lib/types/utils/isa.js 7:0-54
Module not found: Error: Can't resolve 'cljs/metabase.types' in '/Users/work/Metabase/metabase/frontend/src/metabase-lib/types/utils'
 @ ./static-viz/lib/format.ts 16:0-97 40:13-19 44:13-19 48:42-54 51:42-50 59:43-52
 @ ./static-viz/components/RowChart/RowChart.tsx 9:0-126 35:29-58 43:24-48 47:23-42
 @ ./static-viz/components/RowChart/index.ts 2:0-37 2:0-37
 @ ./static-viz/containers/StaticChart/StaticChart.tsx 8:0-63 44:31-39
 @ ./static-viz/containers/StaticChart/index.js 1:0-40 1:0-40
 @ ./static-viz/index.js 2:0-51 5:64-75

ERROR in ./lib/formatting/currency.js 1:0-62
Module not found: Error: Can't resolve 'cljs/metabase.shared.util.currency' in '/Users/work/Metabase/metabase/frontend/src/metabase/lib/formatting'
 @ ./lib/formatting/numbers.tsx 21:0-73 106:59-76 143:83-107
 @ ./static-viz/lib/numbers.ts 7:0-82 23:34-49
 @ ./static-viz/components/CategoricalDonutChart/CategoricalDonutChart.jsx 23:0-64 127:26-39 141:20-32
 @ ./static-viz/components/CategoricalDonutChart/index.js 1:0-50 1:0-50
 @ ./static-viz/containers/StaticChart/StaticChart.tsx 12:0-89 35:31-52
 @ ./static-viz/containers/StaticChart/index.js 1:0-40 1:0-40
 @ ./static-viz/index.js 2:0-51 5:64-75

4 errors have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.85.0 compiled with 4 errors and 1 warning in 1414 ms
```

### Information about your Metabase installation

`master` commit `4b27f570db`

### Severity

Makes it difficult to develop static viz

--- a/package.json
+++ b/package.json
@@ -328,7 +328,7 @@
     "build-release": "yarn build-release:cljs && yarn build-release:js",
     "build-stats": "yarn && webpack --json > stats.json",
     "build-shared": "yarn && webpack --config webpack.shared.config.js",
-    "build-static-viz": "yarn && yarn build-release:cljs && webpack --config webpack.static-viz.config.js",
+    "build-static-viz": "yarn && yarn build-release:cljs && WEBPACK_BUNDLE=production webpack --config webpack.static-viz.config.js",
     "build-static-viz:watch": "webpack --config webpack.static-viz.config.js --watch",
     "precommit": "lint-staged",
     "preinstall": "echo $npm_execpath | grep -q yarn || echo '\\033[0;33mSorry, npm is not supported. Please use Yarn (https://yarnpkg.com/).\\033[0m'",

--- a/webpack.static-viz.config.js
+++ b/webpack.static-viz.config.js
@@ -1,12 +1,16 @@
 const SRC_PATH = __dirname + "/frontend/src/metabase";
 const BUILD_PATH = __dirname + "/resources/frontend_client";
 const CLJS_SRC_PATH = __dirname + "/frontend/src/cljs_release";
+const CLJS_SRC_PATH_DEV = __dirname + "/frontend/src/cljs";
 const LIB_SRC_PATH = __dirname + "/frontend/src/metabase-lib";
 const TYPES_SRC_PATH = __dirname + "/frontend/src/metabase-types";
 
 const BABEL_CONFIG = {
   cacheDirectory: process.env.BABEL_DISABLE_CACHE ? null : ".babel_cache",
 };
+
+const WEBPACK_BUNDLE = process.env.WEBPACK_BUNDLE || "development";
+const devMode = WEBPACK_BUNDLE !== "production";
 
 module.exports = env => {
   const shouldDisableMinimization = env.WEBPACK_WATCH === true;
@@ -47,7 +51,7 @@ module.exports = env => {
       extensions: [".webpack.js", ".web.js", ".js", ".jsx", ".ts", ".tsx"],
       alias: {
         metabase: SRC_PATH,
-        cljs: CLJS_SRC_PATH,
+        cljs: devMode ? CLJS_SRC_PATH_DEV : CLJS_SRC_PATH,
         "metabase-lib": LIB_SRC_PATH,
         "metabase-types": TYPES_SRC_PATH,
       },

